src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -179,6 +180,19 @@ void SolverInterfaceImpl::configure(
   configure(config.getSolverInterfaceConfiguration());
 }
 
+namespace {
+/// Keeps track of SolverInterface initalizations and returns a suffix for the event timings file
+std::string eventSuffix()
+{
+  static int run{0};
+  ++run;
+  if (run == 1) {
+    return "";
+  }
+  return std::to_string(run);
```

Comment:
```suggestion
  return std::to_string("-" + run);
```
? No strong opinion.

Reply:
EventUtils handle this kind of formatting.